tools/validate_spec.py

```diff
@@ -1,16 +1,20 @@
 #!/usr/bin/env -S python3 -O
```

Comment:
Why is this one test so extremely slow?

```
53.75s call     tests/test_validation_tool.py::test_cli_not_regular_file
7.24s call     tests/test_validation_tool.py::test_validation_negative_full_output
6.73s call     tests/test_validation_tool.py::test_coverage_threshold_missed
6.65s call     tests/test_validation_tool.py::test_validation_positive_full_output
```

Reply:
For this test, the `--no-verification` flag was missing. Fixed in 2072060a.